Common/brushed_dc.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
```

Comment:
Please put this in its own branch/commit. This should not be in this pull request.